lib/cisco_node_utils/cmd_ref/README_YAML.md

```diff
@@ -469,6 +472,23 @@ By convention, a `default_value` of `''` (empty string) represents a configurabl
 
 `config_get()` will return the defined `default_value` if the defined `config_get_token` does not match anything on the node. Normally this is desirable behavior, but you can use [`auto_default`](#auto_default) to change this behavior if needed.
 
+### `default_only`
+
+Some attributes may be hard-coded on certain platforms in such a way that they have a meaningful `default_value` but no relevant `config_get_token` or `config_set` behavior. On such platforms, the key `default_only: true` can be set, which causes the `config_get()` API to always return the default value and `config_set()` to raise a `Cisco::UnsupportedError`.
```

Comment:
If I understand correctly, you're saying the following example YAML is possible:

```
feature_foo:
  default_only:
  default_value: 10
  /PLATFORM/:
    default_only: false
    [snip]
```

Sounds reasonable to me.